src/main/java/cl/transbank/pos/responses/KeysResponse.java

```diff
@@ -29,6 +29,13 @@ public String getTerminalId() {
         return terminalId;
     }
 
+    public KeysResponse( int fc, int rc, long cc, String ti) {
```

Comment:
You need to be more specific with the variables in the method signature, since not everyone will understand the abbreviation.

Reply:
It is true, the truth I don't know what they mean. @floydnunez can you help us with its meaning?

Reply:
You have the meaning in the assignation. Just repeat it.